udata/api_fields.py

```diff
@@ -293,6 +347,8 @@ def patch(obj, request):
     Patch the object with the data from the request.
     Only fields decorated with the `field()` decorator will be read (and not readonly).
     """
+    from udata.mongo.engine import db
```

Comment:
This is done a lot inside the app to avoid circular dependencies. When importing at the top of a file, the import is done at startup and can cause circular dependencies, if it's inside a function at runtime everything is often imported and there is less problems.